test/system/035-logs.bats

```diff
@@ -324,16 +324,16 @@ function _log_test_follow_since() {
 
     # Now do the same with a running container to check #16950.
     run_podman ${events_backend} run --log-driver=$driver --name $cname -d $IMAGE \
-        sh -c "sleep 1; while :; do echo $content && sleep 5; done"
+        sh -c "sleep 1; while :; do echo $content && sleep 1; done"
 
     # sleep is required to make sure the podman event backend no longer sees the start event in the log
     # This value must be greater or equal than the value given in --since below
     sleep 0.2
 
     # Make sure podman logs actually follows by giving a low timeout and check that the command times out
     PODMAN_TIMEOUT=3 run_podman 124 ${events_backend} logs --since 0.1s -f $cname
-    assert "$output" =~ "^$content
-timeout: sending signal TERM to command.*" "logs --since -f on running container works"
+    assert "$output" =~ "${content}
+timeout: sending signal TERM to command.*\$" "logs --since -f on running container works"
```

Comment:
Sorry, I wasn't clear.

You added a dollar sign. It is a NOP, it does not do anything. So when the next programmer comes to look at this code some day, she or he will be confused by it, wondering why it exists. This will cost them time.

Please remove it, making it back like it was. Even better, remove the `.*` which is also useless (that one is my fault). Make it just `...signal TERM to command"`

Reply:
Thanks, I get it now.